test_schemas/en/test_new_routing_checkbox_contains.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
These schemas need more details (UI), I don't know what I should be selecting to get to the right/wrong page.  In this instance I selected Malta and I ended up on the submission page not knowing if that was right or wrong, I think this one needs another interstitial page. I also think a simple description about what should be entered and what happens is needed

Reply:
Ok, now they have detailed descriptions very similar to those in @berroar's latest PR, also some updated instructions and updated functional tests. Thanks!